### PR TITLE
Added missing IIIT Delhi faculty

### PR DESCRIPTION
**Pull request by siddjai**
Added the following professors, all of whom can advise CS PhD students:
[removed] Abir Das, https://iiitd.ac.in/abir
[removed] Angshul Majumdar, http://faculty.iiitd.ac.in/~angshul/
Anubha Gupta, http://faculty.iiitd.ac.in/~anubha
Gajendra P. S. Raghava, http://webs.iiitd.edu.in/raghava
Ganesh Bagler, http://faculty.iiitd.ac.in/~bagler
[removed] P. B. Sujit, https://iiitd.ac.in/sujit
Syamantak Das, http://faculty.iiitd.ac.in/~syamantak

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -89,6 +89,7 @@ Abhishek Mishra,BITS Pilani,http://www.bits-pilani.ac.in/pilani/abhishekmisra/pr
 Abhishek Sarkar,IIIT Hyderabad,https://www.iiit.ac.in/people/faculty/abhishek.sarkar,SV0gEjQAAAAJ
 Abhishek Srivastava,IIT Indore,http://cse.iiti.ac.in/faculty.html,8DUQgIIAAAAJ
 Abilio Lucena,UFRJ,http://www.cos.ufrj.br/index.php/pt-BR/pessoas/details/18/2201-abiliolucena,_YS2DcIAAAAJ
+Abir Das, IIIT Delhi,https://iiitd.ac.in/abir,L4yEk2UAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Remove space after comma.

File: csrankings.csv [hunk 2]
```diff
@@ -876,6 +878,7 @@ Angelika Steger,ETH Zurich,http://www.as.inf.ethz.ch/people/prof,Kzeoyq8AAAAJ
 Angelo C. Morzenti,Politecnico di Milano,http://home.deib.polimi.it/morzenti,NOSCHOLARPAGE
 Angelo Marcelli,University of Salerno,http://docenti.unisa.it/003780/home,NOSCHOLARPAGE
 Angelos Stavrou,George Mason University,https://cs.gmu.edu/~astavrou,hnuQ6PMAAAAJ
+Angshul Majumdar,IIIT Delhi,http://faculty.iiitd.ac.in/~angshul/,nwNIkAUAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Not in CS. Please verify that all added faculty are in CS. Also, please remove trailing slashes from URLs.

**Comment by emeryberger** [comment]
Closing for lack of response.

**Comment by peey** [comment]
@emeryberger what's the issue with this PR? It's not clear. This change fits the following description from `CONTRIBUTING.md`:

**Adding one or more faculty members (including an entire department)**

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

If there is no issue, could this be re-opened / merged?

**Comment by emeryberger** [comment]
I accidentally did not submit my review; sorry.

**Comment by siddjai** [comment]
@emeryberger Changes made, please review.

**Pull request merged**